### DISCUSSION TITLE
Text Size property for the divider

### DISCUSSION THREAD

**Discussion by ArianYN** [sqlpage/SQLPage#617] [Ideas]
Would it be possible to get a property for the 'Divider' Component to change the text size in the 'contents' property?

**Comment by lovasoa** [comment]
Yes ! Do you want to make a small pull request to https://github.com/lovasoa/SQLpage/blob/main/sqlpage/templates/divider.handlebars ?

**Reply by ArianYN** [reply]
I'm not familiar with developing SQLPage, sorry. I was asking if anyone could maybe add the feature :)
